### PR TITLE
Read package metadata from tar.bz2

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #5 
Added `InfoIndex` interface to obtain metadata from the package and first implementation to read info/index.json from tar.bz2 archive. Also, added `java` directory to tests (it was missing).

### CODE REVIEW

File: src/main/java/com/artipie/conda/meta/InfoIndex.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * @since 0.2
+ */
+public interface InfoIndex {
+
+    /**
+     * Conda package metadata info/index.json content as json object.
+     * @return Metadata json
+     * @throws IOException On error
+     */
+    JsonObject json() throws IOException;
```
Comment (genryxy) [inline src/main/java/com/artipie/conda/meta/InfoIndex.java:31]: @olenagerasimova thanks, one question. Why there is returning type just `JsonObject`, not `CompletionStage<JsonObject>`?

Comment (olenagerasimova) [reply src/main/java/com/artipie/conda/meta/InfoIndex.java:31]: @genryxy what is the point to return `CompletionStage`here? All operations here are blocking and sdk api too. To unpack archive I have to work with input streams (there are no other way in java), so in http level I will transform publisher to input stream (using `rio` or simply creating byte array input steam) and create instance of this class to read metadata. This is "the lowest" level.

Comment (genryxy) [reply src/main/java/com/artipie/conda/meta/InfoIndex.java:31]: @olenagerasimova okay, got it

**Review by genryxy — APPROVED** [review]

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**